cocos/core/renderer/scene/submodel.ts

```diff
@@ -371,7 +371,13 @@ export class SubModel {
         // update draw info
         const drawInfo = this._subMesh.drawInfo;
         if (this._inputAssembler && drawInfo) {
-            this._inputAssembler.drawInfo.copy(drawInfo);
+            this._inputAssembler.drawInfo.vertexCount = drawInfo.vertexCount;
```

Comment:
Any problem with copy?

Reply:
`DrawInfo ` has been exported by emscripten, ems value cannot be copied from js drawInfo, so copy here

Reply:
The drawinfo exported by emscripten is not a standard js object?

Reply:
So `this._inputAssembler.drawInfo` and `drawInfo` are different types on WebGPU?
Could we export copy method to JS and implement copy in C++ by emscripten::val ?

Reply:
>The drawinfo exported by emscripten is not a standard js object

No, it's more like a jsb object.
>So this._inputAssembler.drawInfo and drawInfo are different types on WebGPU?

No, they are different. We can meke them the same type, but performance may get affected. Considering `commandBuffer.draw` may be called frequently per frame, to reduce js->c++ data conversion so I decide to export drawInfo. This is just a choice of `performance` or `usage`.

* If `DrawInfo` exported by emscripten, we can fill data and pass it directly to c++ interface in js. If not, we have to pass js object `DrawInfo` wrapped by `emscripten::val`, and then read property one by one and fill c++ `DrawInfo` in native. This may be a performance harzard.

Reply:
Okay, is there any other places which also needs to do the `copy` logic? Shall we add a helper function for this copy purpose instead of writing it in many places?

Reply:
this is the only place I found. `copy` will be added later.